manifests/base/webhook/manifests.yaml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
Can we name this file as `webhooks.yaml`, similar to Katib: https://github.com/kubeflow/katib/blob/master/manifests/v1beta1/components/webhook/webhooks.yaml and other Training Operator deployment manifest name: https://github.com/kubeflow/training-operator/blob/7cc1365c5aac01beef660709a5139321a5e62675/manifests/base/deployment.yaml.

Or kubebuilder sets this name by default ?

Reply:
> Or kubebuilder sets this name by default ?

Yea, this name is default value 😓